src/Instrumentation/Psr6/src/Psr6Instrumentation.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Contrib\Instrumentation\Psr6;
+
+use Composer\InstalledVersions;
+use OpenTelemetry\API\Instrumentation\CachedInstrumentation;
+use OpenTelemetry\API\Trace\Span;
+use OpenTelemetry\API\Trace\SpanBuilderInterface;
+use OpenTelemetry\API\Trace\SpanKind;
+use OpenTelemetry\API\Trace\StatusCode;
+use OpenTelemetry\Context\Context;
+use function OpenTelemetry\Instrumentation\hook;
+use OpenTelemetry\SemConv\TraceAttributes;
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use Throwable;
+
+/**
+ * @psalm-suppress ArgumentTypeCoercion
+ */
+class Psr6Instrumentation
+{
+    public const NAME = 'psr6';
+
+    public static function register(): void
+    {
+        $instrumentation = new CachedInstrumentation(
+            'io.opentelemetry.contrib.php.psr6',
+            InstalledVersions::getVersion('open-telemetry/opentelemetry-auto-psr6')
```

Comment:
```suggestion
            InstalledVersions::getVersion('open-telemetry/opentelemetry-auto-psr6'),
            'https://opentelemetry.io/schemas/1.25.0',
```
(assuming latest semconv, 1.25.0, is the schema you developed against)